### ISSUE TITLE
appConfigPath in Setup is not implemented

### ISSUE THREAD

**Issue by hh4711** [geert-janklaps/cds-launchpad-plugin#12]
Hi,
I have the requirement to start the Launchpad tiles in different subdirectories.

But the appConfigPath with my own config doesn_t work. 

I used this code:
 app.use(handler.setup({theme:'sap_horizon', version: '1.99.0', appConfigPath: '/fiori/appconfig/fioriSandboxConfig'}));
 
 Is it possible to implement. If not, could you please adjust 

in index.js line 149 
                   // App URL
                    const url = `/${element.replace(cds.env.folders.app, '')}/webapp`;
 
my element 'main-app/app/backend' is replaced to /main-app/backend, but this is wrong. It should be: /main-app/app/backend

Thank you in advance.

**Comment by ghost** [comment]
Hi @hh4711 ,

are you using this cds config to address your different folder for your UI5 apps?
```json
  "cds": {
    "folders": {
      "app": "./../app/"
    }
  }
```
cause we also have an issue after the latest patch.
Maybe my solution will fix your issue.
Can you check the changes in your project?

regards
Norbert

**Comment by geert-janklaps** [comment]
This fix is now available in version 2.0.1, thanks for your contribution @nlaenger!
